accumulo/src/main/java/com/yahoo/ycsb/db/ZKProducerConsumer.java

```diff
@@ -1,3 +1,20 @@
+/**
+ * Copyright (c) 2014 YCSB contributors. All rights reserved.
```

Comment:
the Accumulo binding started life in [YCSB++](http://www.pdl.cmu.edu/ycsb++/), we should acknowledge this in the headers.

Also if someone can find a copy of the YCSB++ code that'd be nice. the main repo appears to have succumbed to time.